tests/san/dns_in_san.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Could you please include the script for generating the *.der files from the *.json files?